lib/logflare/sources/source/bigquery/pipeline.ex

```diff
@@ -238,18 +238,6 @@ defmodule Logflare.Sources.Source.BigQuery.Pipeline do
               tesla_response: GenUtils.get_tesla_error_message(response)
             )
         end
-
-      {:error, :emfile = response} ->
-        Logger.error("Stream batch emfile error!", tesla_response: response)
-
-      {:error, :timeout = response} ->
-        Logger.warning("Stream batch timeout error!", tesla_response: response)
-
-      {:error, :checkout_timeout = response} ->
-        Logger.warning("Stream batch checkout_timeout error!", tesla_response: response)
-
-      {:error, response} ->
```

Comment:
Test coverage for this error handling is not high so would need some tests around these logs that are needed for debugging